NYTimes.com.js

```diff
@@ -165,8 +166,8 @@ function scrape(doc, url) {
 	if (metaTags["keywords"]) {
 		var keywords = metaTags["keywords"];
 		newItem.tags = keywords.split(",");
-		for (var i in newItem.tags) {
-			newItem.tags[i] = newItem.tags[i].replace("  ", ", ");
+		for (var t in newItem.tags) {
```

Comment:
This may be a little too much caution, but yes they are in the same function which is the scope for `var`.